distribution.md

```diff
@@ -228,3 +230,89 @@ The Run Image SHOULD contain the following Labels on the image configuration:
+
+A builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting builders SHOULD allow users to configure the build environment for a provided builder.
+
+#### General Requirements
+
+The builder image MUST contain an implementation of the [lifecycle](#lifecycle), and a [build-time](#build-image) environment, and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process.
+
+#### Filesystem
+
+A builder MUST have the following directories/files:
+
+* `/cnb/order.toml` &rarr; As defined in the [platform specification](https://github.com/buildpacks/spec/blob/main/platform.md#ordertoml-toml)
+
+#### Environment Variables
+
+A builder MUST be an extension of a Build Image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+The following environment variables MAY be set on the builder (through the image config's `Env` field):
+
+| Env Variable             | Description                                                                       | Default
+| --------                 | --------                                                                          |-----------
+| `CNB_APP_DIR`            | Application directory of the build environment                                    | `/workspace`
+| `CNB_LAYERS_DIR`         | The directory to create and store `layers` in the build environment               | `/layers`
+| `CNB_PLATFORM_DIR`       | The directory to create and store platform focused files in the build environment | `/platform`
+| `SERVICE_BINDING_ROOT`   | The directory where services are bound                                            | -
+
+
+#### Labels
+
+A builder MUST be an extension of a Build Image, and MUST retain all the specified Labels set on the original build image, as specified in the Build Image specifications.
+
+A builder image MUST contain an implementation of the [lifecycle](#lifecycle), and MUST retain all the specified Labels set on the original Lifecycle image, as specified in the lifecycle distribution specifications.
+
+A builder image MAY contain [buildpacks](#buildpackage), and MAY retain all the specified Labels set on the original buildpackage, as specified in the buildpackage specifications.
+
+The following labels MUST be set in the builder environment (through the image config's `Labels` field):
+
+| Label             | Description
+| --------          | --------
+| `io.buildpacks.builder.metadata`     | A JSON object representing builder metadata
+| `io.buildpacks.buildpack.order`      | A JSON object representing the order of the buildpacks stored on the builder
+
+`io.buildpacks.builder.metadata` (JSON)
+
+```json
+{
+  "description": "<description>",
+  "buildpacks": [
+    {
+      "id": "<buildpack ID>",
+      "version": "<buildpack version>",
+      "homepage": "<buildpack homepage>",
+    }
+  ],
```

Comment:
Is this duplicative of `io.builpacks.buildpackage.layers`. Should we remove this section but require `io.builpacks.buildpackage.layers` on builders?

Reply:
If we go with option 2 of [this comment](https://github.com/buildpacks/spec/pull/271/files#r791819949) then this may also be cleared up.

Reply:
Resolved in 35e5cce6acb83cfa018f61dc0bc90019375b8702